src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -563,6 +590,23 @@ private URI makeGetClientURI(UUID requestId, String pipe) throws URISyntaxExcept
     return builder.build();
   }
 
+  /**
+   * makeOAuthTokenRequestURI - Make the uri for OAuth token request
+   * http://snowflakeURL{:PORT}/oauth/token-request
+   */
+  private URI makeOAuthTokenRequestURI() {
+    try {
+      return new URIBuilder()
+          .setScheme(scheme)
+          .setHost(host)
+          .setPort(port)
+          .setPath(TOKEN_REQUEST_ENDPOINT)
+          .build();
+    } catch (URISyntaxException e) {
+      throw new SFException(e, ErrorCode.MAKE_URI_FAILURE, "Make OAuth token request fail");
+    }
+  }
```

Comment:
Could this be in the OAuth manager and move the `TOKEN_REQUEST_ENDPOINT` constant in there?

Reply:
Fixed